### ISSUE TITLE
cant install module

### ISSUE THREAD

**Issue by tiefenb** [karltaidev/SessionHandlerDBRedis#1]
<img width="1440" alt="bildschirmfoto 2019-01-20 um 19 31 31" src="https://user-images.githubusercontent.com/488163/51443608-9be6d780-1ceb-11e9-815b-1f55cd9c2592.png">

Processwire: 3.0.124
PHP: 7.0.31

**Comment by adrianbj** [comment]
Did you try installing using this class name: SessionHandlerRedis

http://modules.processwire.com/modules/session-handler-redis/

**Comment by adrianbj** [comment]
Sorry, I didn't realize there are two different Redis modules for PW, this one and https://github.com/netcarver/PW-SessionHandlerRedis 

It looks like you are having difference issues with each of the different modules. My mistake :)

**Comment by tiefenb** [comment]
yes - after getting the Issue with SessionHandlerRedis, I tried this module and get an other issue
